### PR TITLE
Cleanup/rename and align domain models

### PR DESCRIPTION
**Pull request by jonasdmoeller**
Cleanup af navngivning og alignment med terraform labels

### CODE REVIEW

File: build/infrastructure/main/func-functionhost.tf [hunk 1]
```diff
@@ -57,8 +57,8 @@ module "func_functionhost" {
     CHARGE_COMMAND_RECEIVED_SUBSCRIPTION_NAME                       = "sbtsub-charges-charge-command-received"
     CHARGE_COMMAND_REJECTED_SUBSCRIPTION_NAME                       = "sbtsub-charges-charge-command-rejected"
     CHARGE_PRICE_COMMAND_RECEIVED_SUBSCRIPTION_NAME                 = "sbtsub-charges-charge-price-command-received"
-    CHARGE_PRICE_REJECTED_SUBSCRIPTION_NAME                         = "sbtsub-charges-charge-price-rejected"
-    CHARGE_PRICE_CONFIRMED_SUBSCRIPTION_NAME                        = "sbtsub-charges-charge-price-confirmed"
+    CHARGE_PRICE_OPERATIONS_REJECTED_SUBSCRIPTION_NAME              = "sbtsub-charges-charge-price-rejected"
+    CHARGE_PRICE_OPERATIONS_CONFIRMED_SUBSCRIPTION_NAME             = "sbtsub-charges-charge-price-confirmed"
```
Comment (HenrikSommer) [inline build/infrastructure/main/func-functionhost.tf:61]: Should we change the subscription names as well?
```suggestion
    CHARGE_PRICE_OPERATIONS_REJECTED_SUBSCRIPTION_NAME              = "sbtsub-charges-charge-price-operations-rejected"
    CHARGE_PRICE_OPERATIONS_CONFIRMED_SUBSCRIPTION_NAME             = "sbtsub-charges-charge-price-operations-confirmed"
```

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/Charges/Acknowledgement/ChargePublisher.cs [hunk 2]
```diff
@@ -39,7 +39,7 @@ public async Task PublishChargeCreatedAsync(ChargeInformationOperationDto charge
             await _messageChargeDispatcher.DispatchAsync(chargeCreatedEvent).ConfigureAwait(false);
         }
 
-        public Task PublishChargeUpdatedAsync(ChargeCommandAcceptedEvent chargeCommandAcceptedEvent)
+        public Task PublishChargeUpdatedAsync(ChargeInformationCommandAcceptedEvent chargeInformationCommandAcceptedEvent)
```
Comment (HenrikSommer) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/Charges/Acknowledgement/ChargePublisher.cs:42]: I suggest we rename `ChargePublisher` and `IChargePublisher`, too.

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/Charges/Handlers/IChargeIntegrationEventsPublisher.cs [hunk 3]
```diff
@@ -19,6 +19,6 @@ namespace GreenEnergyHub.Charges.Application.Charges.Handlers
 {
     public interface IChargeIntegrationEventsPublisher
```
Comment (HenrikSommer) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/Charges/Handlers/IChargeIntegrationEventsPublisher.cs:20]:
```suggestion
    public interface IChargeInformationIntegrationEventsPublisher
```

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/GreenEnergyHub.Charges.Application.csproj [hunk 4]
```diff
@@ -51,4 +51,10 @@ limitations under the License.
     </Compile>
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="GreenEnergyHub.Charges.Domain">
+      <HintPath>..\GreenEnergyHub.Charges.FunctionHost\bin\Debug\net6.0\GreenEnergyHub.Charges.Domain.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+
```
Comment (HenrikSommer) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/GreenEnergyHub.Charges.Application.csproj:59]: Is this necessary?

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/Charges/ChargeIntegrationEventsPublisherEndpoint.cs [hunk 5]
```diff
@@ -24,11 +24,11 @@ namespace GreenEnergyHub.Charges.FunctionHost.Charges
     public class ChargeIntegrationEventsPublisherEndpoint
```
Comment (HenrikSommer) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/Charges/ChargeIntegrationEventsPublisherEndpoint.cs:24]:
```suggestion
    public class ChargeInformationIntegrationEventsPublisherEndpoint
```

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.MessageHub/Models/AvailableOperationReceiptData/ChargePriceCimValidationErrorTextFactory.cs [hunk 6]
```diff
@@ -42,27 +42,27 @@ public class ChargePriceCimValidationErrorTextFactory : ICimValidationErrorTextF
         public string Create(
             ValidationError validationError,
             DocumentDto document,
-            ChargePriceOperationDto operationDto)
+            ChargePriceOperationDto operationDtoDto)
```
Comment (HenrikSommer) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.MessageHub/Models/AvailableOperationReceiptData/ChargePriceCimValidationErrorTextFactory.cs:45]:
```suggestion
            ChargePriceOperationDto operationDto)
```

**Review by HenrikSommer — APPROVED** [review]

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/Charges/Acknowledgement/ChargePublisher.cs (hunk 2 above)
Comment (jonasdmoeller) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/Charges/Acknowledgement/ChargePublisher.cs:42]: This is subject to change if we can make a common interface for D18 and D08 flow, so I would prefer keeping it in mind for later 👍

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/Charges/Handlers/IChargeIntegrationEventsPublisher.cs (hunk 3 above)
Comment (jonasdmoeller) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/Charges/Handlers/IChargeIntegrationEventsPublisher.cs:20]: This is subject to change if we can make a common interface for D18 and D08 flow, so I would prefer keeping it in mind for later 👍

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/GreenEnergyHub.Charges.Application.csproj (hunk 4 above)
Comment (jonasdmoeller) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/GreenEnergyHub.Charges.Application.csproj]: removed

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/MessageHub/OutboxMessageProcessorEndpoint.cs [hunk 7]
```diff
@@ -26,27 +26,27 @@ namespace GreenEnergyHub.Charges.FunctionHost.MessageHub
 {
     public class OutboxMessageProcessorEndpoint
     {
-        private const string FunctionName = nameof(OutboxMessageProcessorEndpoint);
+        public const string FunctionName = nameof(OutboxMessageProcessorEndpoint);
```
[the remark this answers is not in the document]
Comment (jonasdmoeller) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/MessageHub/OutboxMessageProcessorEndpoint.cs]: fixed

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/Charges/ChargePriceCommandReceiverEndpoint.cs [hunk 8]
```diff
@@ -24,7 +24,7 @@ namespace GreenEnergyHub.Charges.FunctionHost.Charges
 {
     public class ChargePriceCommandReceiverEndpoint
     {
-        private const string FunctionName = nameof(ChargePriceCommandReceiverEndpoint);
+        public const string FunctionName = nameof(ChargePriceCommandReceiverEndpoint);
```
[the remark this answers is not in the document]
Comment (jonasdmoeller) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/Charges/ChargePriceCommandReceiverEndpoint.cs]: fixed

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.MessageHub/Models/AvailableOperationReceiptData/AvailableChargePriceOperationRejectionsFactory.cs [hunk 9]
```diff
@@ -69,14 +69,14 @@ public class AvailableChargePriceOperationRejectionsFactory :
         }
 
         private List<AvailableReceiptValidationError> GetReasons(
-            PriceRejectedEvent input,
-            ChargePriceOperationDto operationDto)
+            ChargePriceOperationsRejectedEvent input,
+            ChargePriceOperationDto operationDtoDto)
```
[the remark this answers is not in the document]
Comment (jonasdmoeller) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.MessageHub/Models/AvailableOperationReceiptData/AvailableChargePriceOperationRejectionsFactory.cs]: fixed

**Pull request merged**